### DISCUSSION TITLE
How do I hide chapter from structure when writing articles?

### DISCUSSION THREAD

**Discussion by LordWolfenstein** [James-Yu/LaTeX-Workshop#3407] [Q&A]
I remove `chapter` from `latex-workshop.view.outline.sections` but there is still an extra `.0` in the structure panel.

![image](https://user-images.githubusercontent.com/27807240/182029523-89979333-1d0a-41d3-b882-01f5343369ff.png)

**Comment by James-Yu** [comment]
Did you reload or restart vscode?
